src/app/containers/Metadata/index.test.jsx

```diff
@@ -709,3 +719,64 @@ shouldMatchSnapshot(
     title="BBC Ukrainian"
   />,
 );
+
+describe('apple-itunes-app meta tag', () => {
+  // eslint-disable-next-line react/prop-types
+  const CanonicalCPSAssetInternationalOrigin = ({ service }) => (
+    <MetadataWithContext
+      service={service}
+      bbcOrigin={dotComOrigin}
+      platform="canonical"
+      id="asset-12345678"
+      pageType="STY"
+      pathname={`/${service}/asset-12345678`}
+      {...newsArticleMetadataProps}
+    />
+  );
+
+  const itunesAppIds = [
+    {
+      service: 'arabic',
+      id: 558497376,
+    },
+    {
+      service: 'mundo',
+      id: 515255747,
+    },
+    {
+      service: 'russian',
+      id: 504278066,
+    },
+  ];
+
+  itunesAppIds.forEach(application => {
```

Comment:
Hey test looks good just an opportunity to use the `it.each` feature in jest to make parameterised tests a bit nicer to read like here: https://github.com/bbc/simorgh/blob/latest/src/app/pages/StoryPage/index.test.jsx#L276..L282

Reply:
Done 👍